### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by jackvreeken**
- [x] Add/change test case for outputs and expansion

### CODE REVIEW

File: test/gen_casadi_test.py [hunk 1]
```diff
@@ -658,7 +658,7 @@ def test_delay_for_loop_with_expand_vectors(self):
         ref_model = Model()
 
         def _array_mx(name, n):
-            return np.array([ca.MX.sym("{}[{}]".format(name, i)) for i in range(n)])
+            return np.array([ca.MX.sym("{}[{}]".format(name, i+1)) for i in range(n)])
```
Comment (vdwees) [inline test/gen_casadi_test.py]: should be `i + 1` tho I am not sure we care at this point

File: test/models/Annotations.mo [hunk 2]
```diff
@@ -0,0 +1,11 @@
+model A
+    Real x, y;
```
Comment (vdwees) [inline test/models/Annotations.mo]: Why not include an inline annotation here as well?

Comment (jackvreeken) [reply test/models/Annotations.mo]: I couldn't come up with one that makes sense. I just copied the large complicated annotation from a model that failed.

File: test/gen_casadi_test.py (hunk 1 above)
Comment (jackvreeken) [reply test/gen_casadi_test.py]: We have `i+1` in `model.py`, so in a sense it's consistent. If I'm going to change this, I'd have to change the formatting of those as well.

Comment (vdwees) [reply test/gen_casadi_test.py]: sure, ok. tox will complain tho I think.

File: test/models/Annotations.mo (hunk 2 above)
Comment (vdwees) [reply test/models/Annotations.mo]: fair enough. good for now.

**Pull request merged**